### PR TITLE
update about links

### LINKED ISSUE #13780: [beta] Clicking on "About" in the menu opens the map
#### Issue Description

In any page, clicking Community > About in the navigation bar opens the map page instead. After a few seconds a message "Oops! Something went wrong" appears, but no message is shown in the console.

#### Browser Information

* Browser Name, Version: 56.0.2924.87 (64-bit)
* Operating System: Windows 7, 64bit
* Mobile, Desktop, or Tablet: Desktop

### PR DESCRIPTION
**Pull request by dhcodes**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of freeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #13780
 ... Also part of https://github.com/freeCodeCamp/freeCodeCamp/issues/15474

#### Description

Added the static about page and updated mouse-trap accordingly. Within this PR I also finished cleaning up some remnants of the Store removal/Donate link update from this PR (https://github.com/freeCodeCamp/freeCodeCamp/pull/14961)

### CODE REVIEW

File: client/sagas/mouse-trap-saga.js [hunk 1]
```diff
@@ -26,6 +25,10 @@ export default function mouseTrapSaga(actions$) {
     ...Object.keys(softRedirects)
       .map(key => bindKey(key, () => push(softRedirects[key]))),
     bindKey(
+      'g n a',
+      () => hardGoTo('https://www.freecodecamp.com/about')
```
Comment (ahstro) [inline client/sagas/mouse-trap-saga.js]: Could this use the `aboutUrl` from `constantStrings.json`?

**Review by ahstro — CHANGES_REQUESTED** [review]

File: common/utils/constantStrings.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
   "defaultProfileImage": "https://s3.amazonaws.com/freecodecamp/camper-image-placeholder.png",
+  "aboutUrl": "https://www.freecodecamp.com/about",
```
Comment (systimotic) [inline common/utils/constantStrings.json]: Can you change this to .org?

File: common/app/components/Nav/links.json [hunk 3]
```diff
@@ -30,7 +30,8 @@
       },
       {
         "content": "About",
-        "link": "/about"
+        "link": "https://www.freecodecamp.com/about",
```
Comment (systimotic) [inline common/app/components/Nav/links.json]: Can you change this to .org?

File: client/sagas/mouse-trap-saga.js (hunk 1 above)
Comment (systimotic) [reply client/sagas/mouse-trap-saga.js]: I think that would be a good idea. It does look like there's somehow some duplication going on, that stuff in main.js appears to do the same as what is done here

**Review by systimotic — CHANGES_REQUESTED** [review]
Great work @dhcodes! This is a nice way to fix this problem. We may still need the warning toast as suggested in the issue for some of the other static pages.

**Comment by dhcodes** [comment]
okay, I think this is better off now @systimotic

**Comment by BerkeleyTrue** [comment]
@dhcodes Thanks!

Happy Coding!

**Pull request merged**